dsc/workflows/base/__init__.py

```diff
@@ -357,45 +415,40 @@ def finalize_items(self) -> None:
 
             item_submission.ingest_attempts += 1
 
-            result_message = result_message_map.get(item_submission.item_identifier)
+            result_message = result_message_map.get(item_submission.item_identifier)  # type: ignore[assignment]
```

Comment:
I could not figure out how stop `mypy` from complaining about this line without an invasive and seemingly duplicative check (I assumed it would be covered by line 421 but I guess not) so I added an `#type: ignore`